packages/web-wallet/src/analytics.js

```diff
@@ -0,0 +1,8 @@
+import ReactGA from 'react-ga';
```

Comment:
Let's move analytics-related stuff under `src/utils` directory (like Honeybadger script) and initialize it in the `src/plugins.js` file